solver/src/solver.rs

```diff
@@ -111,6 +113,20 @@ pub fn create(
                 chain_id,
                 fee_discount_factor,
             )),
+            SolverType::Quasimodo => boxed(HttpSolver::new(
```

Comment:
nanonit: Instead of repeating the exact same arguments for 90% of the setup between MIP & Quasimode, you could pull this into a shared lambda or private method (so that we are guaranteed to update both solvers in case we want to change a value)

Reply:
Good idea!